### ISSUE TITLE
A  New Bug When I Update to The lasted Version

### ISSUE THREAD

**Issue by xiongcaichang** [react-native-community/react-native-svg#146]
when I use  Text in react-native-svg  the sentence  can't display when there have  a spaces in the middle of a sentence

what should I do,if it's a bug,I hope you can help to fix 

Thanks

**Comment by magicismight** [comment]
Can you provide some code to reproduce it?

**Comment by xiongcaichang** [comment]
```JavaScript
 <Text key= 'title' fill="black"  x={screen.width/2 -10} y='10' fontSize="15" textAnchor="middle" fill="#292f33">全公司销售趋势</Text>
 <Text key= 'label' fill="#8899a6"  x='40' y='45' fontSize="12" textAnchor="start">{`金额(${chartData.currencyUnit})`}</Text>
```
It will appear at two way which like above

**Comment by magicismight** [comment]
This has been fixed in 5.1.1.
